gapis/api/vulkan/framegraph.go

```diff
@@ -166,6 +212,45 @@ type framegraphInfoHelpers struct {
 	rpInfos  []*renderpassInfo
 	rpInfo   *renderpassInfo
 	currRpId uint64
+	// imageLookup is a lookup table to quickly find images that match a memory
```

Comment:
Very nit: I think that writing "memoization table" can save a lot of word.

Reply:
Memoization is more about lazily computing results, storing them, and then lookup them up next time they are queried. I think lookup table is more accurate here since we populate the entire table upfront, and then we always use it for lookups. I also feel that it's semantics details at this point :)